skimage/segmentation/_chan_vese.py

```diff
@@ -237,4 +240,8 @@ def chan_vese(img, mu, lambda1=1.0, lambda2=1.0, tol=1.0, maxiter=100, dt=1.0,
         old_energy = new_energy
         i += 1
     segmentation = phi > 0
-    return (phi, segmentation, energies)
+    if extended_output:
```

Comment:
Tests have to be updated now :).
By the way, could be a good style to return `(segmentation, phi, energies)`, do you agree?